packages/modules/desktop-viewer-react/src/utilities/MakeCancellable.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Please advise on if this is the correct place for this file. Currently each package has their own where necessary, meaning lots of repeating code. However, I didn't see it fit to export this from a single location, unless it should be exported from the base package.

Reply:
if its the same exact file in all 3 pkgs, i would add and export it in the base (viewer-react) and import accordingly in the web and desktop